homeassistant/components/frontier_silicon/media_player.py

```diff
@@ -215,6 +229,8 @@ def media_image_url(self):
             self._mute = None
             self._media_image_url = None
 
+            self._volume_level = None
+
```

Comment:
Set volume level to None here as well.
```python
self._volume_level = None
```

Reply:
Sorry, you've lost me, it's already set to None here?

Reply:
I have lost myself 😜 I meant `_volume_steps `

Reply:
But looking at it now, it does not make much sense to add it here.